src/blockdev.rs

```diff
@@ -225,6 +225,22 @@ pub fn try_discard_all(file: &mut File) -> Result<bool> {
 ioctl_none!(blkrrpart, 0x12, 95);
 ioctl_write_ptr_bad!(blkdiscard, request_code_none!(0x12, 119), [u64; 2]);
 
+pub fn udev_settle_until_partition_appears(device: &str, label: &str) -> Result<()> {
+    eprint!("Waiting for partition label {} to come online", label);
+    for _ in (0..10).rev() {
```

Comment:
If we're going to do this, we might as well increase the number of iterations.  Timeout leads to global failure anyway, and most systems should return on the first iteration.